app/src/main/res/values/styles.xml

```diff
@@ -94,7 +101,7 @@
 
     <style name="ToolbarTitle">
         <item name="fontFamily">sans-serif</item>
-        <item name="android:lineSpacingExtra">6sp</item>
+        <item name="android:lineSpacingExtra">@dimen/default_line_spacing</item>
```

Comment:
I don't think this should be here. Is it really necessary for the toolbar?

Reply:
I took this from the Zeplin layout. The title is multiline.

Reply:
I did not add this. I just extracted it to a dimension. Maybe @elgatovital can help?